.github/workflows/probot-check-group.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   required-jobs:
     runs-on: ubuntu-latest
     if: github.event_name != 'issue_comment' || contains(github.event.comment.body, '@probot pls')
+    timeout-minutes: 5
```

Comment:
Setting the timeout here won't change anything, will it? (not sure but I guess because it doesn't account for queueing time.)

There are tons of jobs waiting in the queue, and they all are triggered by `check_run`.
https://github.com/Lightning-AI/lightning/actions?query=event%3Acheck_run+is%3Aqueued

Reply:
Just saw #14456! :)

Reply:
Right, this does not change anything with running or queued...